$NetBSD: patch-makefile,v 1.1 2013/07/02 10:30:23 adam Exp $

Don't set compiler, nor compiler flags.
Don't overwrite linker flags.

--- makefile.orig	2013-07-02 10:24:50.000000000 +0000
+++ makefile
@@ -2,12 +2,10 @@
 # Makefile for UNIX - unrar
 
 # Linux using GCC
-CXX=g++
-CXXFLAGS=-O2
 LIBFLAGS=-fPIC
 DEFINES=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -DRAR_SMP
 STRIP=strip
-LDFLAGS=-pthread
+LDFLAGS+=-pthread
 DESTDIR=/usr
 
 # Linux using LCC
